brainkeeper/src/app/app.component.scss

```diff
@@ -0,0 +1,26 @@
+#fab {
+    position: absolute;
+    right: 15px;
+    bottom: 15px;
+}
+
+@import '../styles.scss';
+
```

Comment:
All lines here need to be removed

Reply:
changed with commit: 3c092315bd02c68198ed286b95baee75aafa7339